src/main/webapp/i18n/de/course.json

```
@@ -120,6 +120,7 @@
                 }
             },
             "averageScore": "Punkte im Schnitt",
+            "tutorRating": "Tutor Bewertung",
```

```suggestion
            "tutorRating": "Tutorenbewertung",
```
I’m assuming this a metric on how a specific tutor performed (judging by the English translation ‘tutor rating’)?
If not, and it is some count on how many results a tutor created, it should probably be ‘Tutorenbewertungen’.

Additionally: This translation string does seem to be unused (or at least `Ctrl+F` on this page and `grep` in the source code directory do no yield any results beside here and in the English translation file).

---

your are right, removed this metric but did not remove the translation 👍